repos/fdbt-site/cypress_tests/cypress/support/steps.ts

```diff
@@ -408,35 +420,83 @@ export const editPassengerTypeOtherProductsPage = () => {
     clickElementByText('Back');
 };
 
-export const editStartDateOtherProductsPage = () => {
+export const editPassengerTypeOtherProductsPage = () => {
     clickRandomElementInTable('govuk-table__body', 'product-link');
     getElementById('product-name').should('not.be.empty');
     getElementById('product-status').should('not.be.empty');
     getElementById('fare-type').should('not.be.empty');
+    editPassengerType();
+};
+
+export const editPassengerTypePointToPointPage = () => {
+    cy.get(`[id^="active-products-"]`).each(($element, index) => {
+        if ($element.text() === '1') {
+            clickElementById(`service-link-${index}`);
+            cy.get('[class=govuk-table__body]').find('a').click();
+            getElementById('service-name').should('not.be.empty');
+            getElementById('service-status').should('not.be.empty');
+            editPassengerType();
+        }
+    });
+};
+
+const editStartDate = () => {
     clickElementById('start-date-link');
     clearDates();
     let dateInput = completeProductDateInformationPage();
     cy.get('[id=start-date]').should('have.text', dateInput.startDate);
     clickElementByText('Back');
 };
 
-export const editEndDateOtherProductsPage = () => {
+export const editStartDateOtherProductsPage = () => {
     clickRandomElementInTable('govuk-table__body', 'product-link');
     getElementById('product-name').should('not.be.empty');
     getElementById('product-status').should('not.be.empty');
     getElementById('fare-type').should('not.be.empty');
+    editStartDate();
+};
+
+export const editStartDatePointToPointPage = () => {
+    cy.get(`[id^="active-products-"]`).each(($element, index) => {
+        if ($element.text() === '1') {
```

Comment:
Same question as above, regarding what happens if the active product count isnt 1

Reply:
A single product is made in the before all so there should only be 1 service on the services page with 1 product. If it doesn't find it maybe I want to raise a failed test